app/src/main/java/org/mozilla/fenix/settings/PairFragment.kt

```diff
@@ -25,6 +30,7 @@ import org.mozilla.fenix.ext.showToolbar
 class PairFragment : Fragment(R.layout.fragment_pair), UserInteractionHandler {
 
     private val qrFeature = ViewBoundFeatureWrapper<QrFeature>()
+    private var cameraPermissionsDenied = false
```

Comment:
I noticed this variable is being set in multiple files, is it safe to keep track of its state across all these different files and initialize it to false each time?